bm-persona/Fitness/FitnessViewController.swift

```diff
@@ -14,8 +14,27 @@ fileprivate let kCardPadding: UIEdgeInsets = UIEdgeInsets(top: 16, left: 16, bot
 fileprivate let kViewMargin: CGFloat = 16
 fileprivate let kTodayClassesHeight: CGFloat = 300
 
-class FitnessViewController: UIViewController {
-    
+class FitnessViewController: UIViewController, SearchDrawerViewDelegate {
```

Comment:
could be moved to a future time, but should we add some sort of n/a on the open times card? right now, the gyms have no open times data (all blank) and it looks kinda weird. could also just not show it at all. I think there's a nil check but no empty check

Reply:
Good point - added an empty check when parsing.